lib/diagnostic.rb

```diff
@@ -26,7 +26,10 @@
 # be writable.
 
 ##
-# your response here
+class Person
+  def initialize(name, age, location)
+  end
+end
 ##
 
```

Comment:
.5
You're half way there! Remember what goes inside the initialize method?
Hint: `@name = name`